### DISCUSSION TITLE
Hi everyone! New Github Education student here

### DISCUSSION THREAD

**Discussion by xarielah** [community/community#71517] [GitHub Education]
### Hi everyone,

Hi there! 

Its a first time for me here, I'm a 1st year student of Computer Sciences.
I'm a self taught junior web developer developing with javascript, typescript, html, css, react (vite, next), express, nest, mongodb, a little with postgresql, also started learning some Rust, following the lord "ThePrimagen" if anyone knows him.
I'm interested in expending my horizons to become a software engineer.

Any recommendation on how to use this amazing program to reach my goals and more?

Thanks!

**Comment by Piero24** [comment]
@ligmahummus start by learning about YouTube and various guides on github on how to use github, create repos, make commits, pull requests etc... after getting the hang of it you will be able to contribute to other people's repositories. This is the main engine of girhub
